### ISSUE TITLE
IHMValidation update cron job doesn't work on production instance

### ISSUE THREAD

**Issue by aozalevsky** [informatics-isi-edu/protein-database#207]
Apparently, after the re-deployment of the production instance, IHMValidation code wasn't updated.

 `/root/ihm_validation_checkout.sh` (the update script) had the right tag  "20240320"
but the code wasn't updated after re-deployment and was showing "20240315" tag. 

After i called it manually, the correct revision was pulled to the server.

**Comment by svoinea** [comment]
@aozalevsky The `production` is`NOT` supposed to do automatically updates like the `dev` that has `crontab` scripts for the daily updates.

The `production` works with tags.
Each time a new `tag` is provided, we run manually the updates.

You will see on the `/root` directory:
 - the `www-update.sh` script that has tags for `derivapy`, `pdb` backend, `python_ihm` and `ihm_validation`
 - the `ihm_validation_checkout.sh` script (a subset of the `www-update.sh` script) that has tag just for `ihm_validation`

In the case a new tag is provided for `derivapy`, `pdb` backend, `python_ihm` or `ihm_validation`, the updated `www-update.sh` script will be run manually.

In the case a new tag is provided just for the `ihm_validation`, the updated `ihm_validation_checkout.sh` script will be run manually.

**Comment by aozalevsky** [comment]
@svoinea The problem is that the tag from `/root/ihm_validation_checkout.sh` wasn't pulled during the redeployment.

**Comment by svoinea** [comment]
@aozalevsky The explanation is that for the redeployment from scratch, I was running `/root/www-update.sh` that had:
```
    "python_ihm_validation 20240315"
```
I should have updated the tag before running it.

**Comment by aozalevsky** [comment]
@svoinea just a reminder to check the tag (the current is `v1.2`) for the upcoming redeployment (with pdb accession codes).
```
commit 5656a5d0bf669dabede3d82de08a6008d81c7044 (HEAD, tag: v1.2, tag: 20240605)
Author: Arthur Zalevsky <aozalevsky@gmail.com>
Date:   Wed Jun 5 20:30:35 2024 -0700

    update ID rendering for summary validation
```

**Comment by svoinea** [comment]
@aozalevsky Should the new tag be applied on production now or at the redeployment?

**Comment by hongsudt** [comment]
Let's update during the redeployment. Don't do anything on production since
it is already working.

**Comment by aozalevsky** [comment]
@svoinea, it's only for the redeployment; no need to update before the redeployment.

**Comment by aozalevsky** [comment]
@svoinea it seems we had a bit of a miscommunication. in https://github.com/informatics-isi-edu/protein-database/issues/207#issuecomment-2176703222 i said the tag should be `v1.2`, but on production, it was set to:

```
"python_ihm_validation 20240605"
```

Because we found and fixed a minor last-minute issue (and I moved the `v1.2` tag to the updated release), production became out of sync with dev. 

I've pulled the correct version and updated the script. Hope we can do better next time with the 2.0 deployment.
